### ISSUE TITLE
Training with raw data

### ISSUE THREAD

**Issue by tejasvnair** [mgharbi/demosaicnet_caffe#3]
Hi,

Is it possible to train the network with grayscale bayer input(.tiff)? I wonder if the training has to be done on demosaicked input?

Thanks!

**Comment by mgharbi** [comment]
I agree that pre-demosaicked training data is not ideal. But for this, you would still need some sort of ground truth RGB image (e.g. a scan of film photograph or some tri-CCD sensor data or the like). Or some alternative way of measuring the output's faithfulness/quality.

**Comment by tejasvnair** [comment]
Thanks!

Can you please share the solver state for the pre-trained bayer_noise model? Or mention the final learning rate? I am trying to fine tune the network.

**Comment by mgharbi** [comment]
Hi,

I unfortunately discarded the training files. If I recall correctly the learning schedule was 10e-4 for 1e6 iterations, then 1e-5

Best,

Michael

**Comment by tejasvnair** [comment]
Thanks again! Do you remember by any chance the total number of iterations?

**Comment by mgharbi** [comment]
I believe 1e6 iterations at lr = 1e-5 as well
